### PR TITLE
Update unicode-symbols extension

### PR DESCRIPTION
**Pull request by xilopaint**
## Description

- Updated the paste action to dynamically show the name and icon of the frontmost application.

## Screenshot

unicode-symbols-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/unicode-symbols/src/components/CharacterActionPanel.tsx [hunk 1]
```diff
@@ -19,8 +20,9 @@ export const CharacterActionPanel = ({ item }: { item: Character }) => {
     <ActionPanel>
       <ActionPanel.Section title="Unicode">
         <Action.Paste
-          title="Paste Character in Active App"
+          title={`Paste Article to ${frontmostApp?.name || "Active App"}`}
```
Comment (j3lte) [inline extensions/unicode-symbols/src/components/CharacterActionPanel.tsx:23]: Why is this renamed to 'Article'? This should be Character.

**Review by j3lte — CHANGES_REQUESTED** [review]

File: extensions/unicode-symbols/src/components/CharacterActionPanel.tsx (hunk 1 above)
Comment (xilopaint) [reply extensions/unicode-symbols/src/components/CharacterActionPanel.tsx:23]: Oh, sorry! It was a typo. I'm fixing now. Thanks for flagging.

Comment (xilopaint) [reply extensions/unicode-symbols/src/components/CharacterActionPanel.tsx]: Done.

**Review by j3lte — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥
